repository-data/development/src/main/resources/hcm-content/content/documents/corporate-website/publication-system/acceptance-tests.yaml

```diff
@@ -1,5 +1,137 @@
 ---
 /content/documents/corporate-website/publication-system/acceptance-tests:
+  /indicator-documents:
+    /faceted-search-dummy-document:
+      /faceted-search-dummy-document[1]:
+        common:FullTaxonomy:
+        - clinical-indicators
+        - nhs-outcomes-framework
+        - compendium
+        - clinical-commissioning-groups-outcomes-framework
+        - team
+        - adult-social-outcomes-framework--ascof-
+        - summary-hospital-level-mortality-indicator--shmi-
+        common:searchable: true
+        hippo:availability: []
+        hippostd:holder: admin
```

Comment:
Could we remove this line? It makes the document open for editing by default whenever one logs into CMS as admin.

Reply:
:+1: